### ISSUE TITLE
IRepositoriesClient.CommitStatus -> IRepositoriesClient.Status

### ISSUE THREAD

**Issue by shiftkey** [octokit/octokit.net#1034]
A naming thing: https://developer.github.com/v3/repos/statuses/

```csharp
var statuses = await github.Repository.CommitStatus.GetAll("octokit", "octokit.net", sha);
```

We should mark this property as obsolete and introduce a new property, named `Status`:

```csharp
var statuses = await github.Repository.Status.GetAll("octokit", "octokit.net", sha);
```

**Comment by RobPethick** [comment]
I'd like to work on this one

**Comment by M-Zuber** [comment]
Have fun, you might want to look #1042 for inspiration :grinning:

**Comment by asizikov** [comment]
@shiftkey Am I right: this issue was already fixed by #1043

**Comment by shiftkey** [comment]
@asizikov yes - another one that missed the "auto-close on PR" boat! Thanks for reminding me!
